### ISSUE TITLE
Uploading from retina downscale option - no happy medium?

### ISSUE THREAD

**Issue by chrisjohnson** [mileswd/mac2imgur#105]
I'm uploading from my retina display, and I either get this:

Without downscale: https://i.imgur.com/edPuoeF.png
With downscale: https://i.imgur.com/MktI4FJ.png

The first, when opened back up, is double sized on all displays that I try on. The second one is blurry (is there no way to downscale without blurring it?), but the correct size.

Am I doing something wrong? Is this just something all retina display screenshots are subject to? Seems weird that all downscaled images are fuzzy

**Comment by ahmadawais** [comment]
@chrisjohnson second one should not be blurry for those without retina screens.

**Comment by chrisjohnson** [comment]
Right, and I'm on retina and it's showing as blurry for me, I was wondering how we can get it to behave nicely for everybody.
